src/include/sof/audio/pcm_converter.h

```diff
@@ -28,8 +28,9 @@ struct comp_buffer;
 
 #if XCHAL_HAVE_HIFI3
 
-#undef PCM_CONVERTER_GENERIC
-#define PCM_CONVERTER_HIFI3
+/* WORKAROUND: disable HIFI3 PCM conversion due to stability issues */
+// #undef PCM_CONVERTER_GENERIC
+// #define PCM_CONVERTER_HIFI3
```

Comment:
It may be a workaround and a hack, but I still don't like that it's using `//` style comments. Not blocking on it though.